modules/ogc/pom.xml

```diff
@@ -77,13 +77,6 @@
           </additionalProjectnatures>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Same here, except that here I'm sure we don't want to touch, all the code in these modules is auto-generated.